### PR TITLE
fix: externalAddr scheme can be set by user

### LINKED ISSUE #747: externalAddr scheme can be set by user

### PR DESCRIPTION
**Pull request by DRK3**
Closes #747

Removed the hardcoded "http://" scheme from Inbound Endpoint.

Signed-off-by: Derek Trider <derek.trider@securekey.com>

### CODE REVIEW

**Comment by troyronda** [comment]
missing semantic prefix in title.

File: pkg/didcomm/transport/http/inbound.go [hunk 1]
```diff
@@ -157,5 +157,5 @@ func (i *Inbound) Stop() error {
 func (i *Inbound) Endpoint() string {
 	// return http prefix as framework only supports http
 	// TODO https://github.com/hyperledger/aries-framework-go/issues/747 configure scheme for external address
```
Comment (rolsonquadras) [inline pkg/didcomm/transport/http/inbound.go]: We can remove this TODO.

**Comment by DRK3** [comment]
> missing semantic prefix in title.

Done!

File: pkg/didcomm/transport/http/inbound.go (hunk 1 above)
Comment (DRK3) [reply pkg/didcomm/transport/http/inbound.go]: done

**Pull request merged**